packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/google-calendar/services/google-calendar-get-events.service.ts

```diff
@@ -90,4 +77,34 @@ export class GoogleCalendarGetEventsService {
       nextSyncCursor: nextSyncToken || '',
     };
   }
+
+  private handleError(error: GaxiosError) {
+    if (
+      error.code &&
+      [
+        'ECONNRESET',
+        'ENOTFOUND',
+        'ECONNABORTED',
+        'ETIMEDOUT',
+        'ERR_NETWORK',
+      ].includes(error.code)
+    ) {
+      throw parseGaxiosError(error);
+    }
+    if (error.response?.status !== 410) {
+      const googleCalendarError: GoogleCalendarError = {
```

Comment:
lastly, why do we do this construction here and not in parseGoogleCalendarError function?